assets/course-theme/blocks/lesson-blocks/index.js

```diff
@@ -212,14 +212,12 @@ export default [
 		apiVersion: 2,
 		edit: function EditPageActions() {
 			const blockProps = useBlockProps( {
-				className: 'sensei-quiz-pagination',
+				className: 'sensei-course-theme__post-pagination',
```

Comment:
Maybe we can think about removing this customs classes and using the generated by the block

Reply:
Yeah, I agree. I don't understand the reason for adding additional custom classes. Although I'm not fixing it here, I'll keep it in mind for future PRs.